### DISCUSSION TITLE
RF messages

### DISCUSSION THREAD

**Discussion by AlexandreUSA** [Dilbert66/esphome-vistaECP#116] [General]
Hi, I am trying to understand what do the RF messages do. I added some devices serials, loop, etc info the yaml file but I am not sure how to use it. I only see the serial numbers in the RF messages in HA. I read the ReadMe but I am still missing the point. Thanks.

**Comment by Dilbert66** [comment]
If you have the corrrect serial and loop numbers entered, the system will use this information to detect open/close of those rf zones.  It won't depend on the timeout so it will respond immediately.  With the vista, there normally is no close event so we have to depend on not seeing fault messages for a timeout period.  Using the RF zone info, we can detect the close event.

**Comment by AlexandreUSA** [comment]
Thanks. That makes sense.
